bucket/curl.json

```diff
@@ -7,12 +7,12 @@
         "64bit": {
             "url": "https://curl.haxx.se/windows/dl-7.83.1_2/curl-7.83.1_2-win64-mingw.tar.xz",
             "hash": "5e89a7b2eeef4e3f156c5cceeb4c343bd412f711d8aa8a50369a0c757ec05114",
-            "extract_dir": "curl-7.83.1-win64-mingw"
+            "extract_dir": "curl-7.83.1_2-win64-mingw"
```

Comment:
The bot has updated to `1_3`, so whilst this fix works, it needs updating to match the latest commit (a9e7759e1435ba4d46e063ab78cf2649bc8fa703).

Reply:
I have added another pull request with the latest change in it. Thanks!